moderntimeline.dtx

```diff
@@ -62,6 +62,7 @@
 %   Right brace   \}     Tilde         \~}
 %
 %
+% ^^A \changes{0.10}{2015/08/31}{Add support to specify label for end date, thanks to ypid and Astrinus}
```

Comment:
No typo, it is a comment in the comment as this version has not been released yet. I wanted to prepare the changelog for you  :wink: